src/gui/properties/peerlistsortmodel.h

```diff
@@ -31,29 +31,35 @@
 #ifndef PEERLISTSORTMODEL_H
 #define PEERLISTSORTMODEL_H
 
-#include <QStringList>
 #include <QSortFilterProxyModel>
+#include <QStringList>
+
 #include "peerlistdelegate.h"
 
-class PeerListSortModel : public QSortFilterProxyModel {
-  Q_OBJECT
+class PeerListSortModel: public QSortFilterProxyModel
+{
+    Q_OBJECT
 
 public:
-  PeerListSortModel(QObject *parent = 0) : QSortFilterProxyModel(parent) {}
+    PeerListSortModel(QObject *parent = 0)
+        : QSortFilterProxyModel(parent)
+    {
+    }
 
 protected:
-  bool lessThan(const QModelIndex &left, const QModelIndex &right) const {
-  switch (sortColumn()) {
-  case PeerListDelegate::IP:
-  case PeerListDelegate::CLIENT: {
-    QString vL = left.data().toString();
-    QString vR = right.data().toString();
-    return Utils::String::naturalCompareCaseInsensitive(vL, vR);
-  }
-  };
-
-  return QSortFilterProxyModel::lessThan(left, right);
-  }
+    bool lessThan(const QModelIndex &left, const QModelIndex &right) const
+    {
+        switch (sortColumn()) {
+        case PeerListDelegate::IP:
+        case PeerListDelegate::CLIENT: {
+            QString vL = left.data().toString();
```

Comment:
Wrong indentation.
Can be fixed by removing redundant variables and curly braces.